pkg/bootstrap/bootstrap.go

```diff
@@ -205,7 +205,7 @@ func getConfig(bsID, bsKey, bsSvrURL string, skipTLS bool, logger log.Logger) (d
 		return deviceConfig{}, err
 	}
 
-	req.Header.Add("Authorization", bsKey)
+	req.Header.Add("Authorization", "Thing "+bsKey)
```

Comment:
@dborovcanin two questions: first should we use `+` for string concatenation rather than `sprintf()` (which I find more readable), and second - should we declare this string variable, or leave it here (again for readability)?

Reply:
Concatenation does not make much difference for short strings, so I'd keep `Sprintf`. I prefer literal over constant for readability, but **only if we don't reuse it anywhere else**. Otherwise, the constant is preferred.